jscomp/core/j.ml

```diff
@@ -172,7 +172,8 @@ and expression_desc =
   (* literally raw JS code 
   *)
   | Array of expression list * mutable_flag
-  | Optional_block of expression 
+  | Optional_block of expression * bool 
+    (* [true] means [identity] *)
```

Comment:
It's not clear what `identity` means here. I guess this should be described as `not nested`.
Or even use `| NotNested | PossiblyNested`.

Reply:
yes, identity means it is a nop, it is still preserved to help cancel unbox/box operations.
I will do a clean up later, thanks for the comment